pkgs/applications/version-management/gitkraken/default.nix

```diff
@@ -127,10 +130,23 @@ let
         patchelf --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" $file
       done
 
-      for file in $(find . -type f \( -name \*.node -o -name ${pname} -o -name \*.so\* \) ); do
+      for file in $(find . -type f \( -name \*.node -o -name ${pname} -o -name git -o -name git-\* -o -name scalar -o -name \*.so\* \) ); do
         patchelf --set-rpath ${libPath}:$out/share/${pname} $file || true
       done
       popd
+
+      pushd $out/share/${pname}/resources/app.asar.unpacked/git
+      for dir in bin lib libexec share; do
```

Comment:
this seems like overkill, 
I think this was done  because bundle git had some directory that nix didn't have.

do we know what is that directory is used for ?

Reply:
This is the original Gitkraken `git` directory tree:
```
.
├── bin
├── libexec
├── share
└── ssl
```

This is the nixpkgs `git` directory tree:
```
/nix/store/zcpq1fivdd131gshg58m4cyq7fwqr3j8-git-2.42.0
├── bin
├── lib
├── libexec
└── share
```

As you can see, the `ssl` directory is not to be found in the nixpkgs `git` tree.

I first tried to just `rm -rf $out/share/${pname}/resources/app.asar.unpacked/git` then `ln` to the nixpkgs `git`, but then it was not possible to push to any repository from Gitkraken due to the following SSL error:
```
Push gitkraken-bundled-git: fatal: unable to access 'https://github.com/nicolas-goudry/nixpkgs.git/': error setting certificate file: /nix/store/l02yd3ssx9qyfljxf0m5yzyw7gyq1g88-gitkraken-9.10.0/share/gitkraken/resources/app.asar.unpacked/git/ssl/cacert.pem
```

I tried to find this request of the `cacert.pem` file in the code without success.

In the end, I just resorted to remove all the subdirectories of the original `git` directory to replace them with the ones from nixpkgs `git`, leaving the original `ssl` directory as-is. And it worked.

Reply:
looking at this 
![image](https://github.com/NixOS/nixpkgs/assets/12160/bbdd075f-7c02-46ac-8d96-54406bf48315)

from `app.asar` you should be able to set `GIT_SSL_CAINFO` to point to cacert from `nixpkgs`

unless they have something different in their CA chain, we should be using the one from nixpkgs

Reply:
I overlooked this, it should be fine now.